patchAPI/example/README.md

```diff
@@ -0,0 +1,18 @@
+To build you need to set the three environment variables `DYNINST`, `BOOST`, and
+`TBB` to the install directories of these three dependencies.
+
+
+For example, if you `spack` installed `dyninst` you could do:
+
+
+```
+export DYNINST=`spack location --install-dir dyninst`
```

Comment:
What are these being used for? Do we need one for _all_ dependencies (e.g., elfutils, libiberty, etc.)?

Reply:
I think without your RPATH fixes, users will have to set these environment variables to specify where to find the installed dependencies. And depending on what are pre-installed on the system, you may need to specify paths to find BOOST, TBB, ELFUTILS, and so on.

I don't think it is worthwhile to tweak these environment variables right now. After your CMake & RPath fixes, we can clear the README file.